src/main/scala/edu/ie3/simona/service/primary/PrimaryServiceWorker.scala

```diff
@@ -270,14 +264,52 @@ object PrimaryServiceWorker extends SimonaService {
 
             Success(initializedStateData, maybeNextTick)
 
-          case (Some(tick), _) if tick > 0L =>
-            /* The start of the data needs to be at the first tick of the simulation. */
+          case (Some(tick), None) if tick > 0L =>
+            /* No data for the first tick or before, but the start of the data needs to be at the first tick of the simulation. */
             Failure(
               new ServiceRegistrationException(
                 s"The data for the timeseries '${source.getTimeSeries.getUuid}' starts after the start of this simulation (tick: $tick)! This is not allowed!"
               )
             )
 
+          case (Some(_), Some(value)) =>
+            /* We have data before and after the start of the simulation, but not at tick 0 */
+            log.debug(
+              s"No data at the start of the simulation. Use last know data for tick: ${value.toTick}"
+            )
+
+            /* Set up the state data. */
+            val initializedStateData =
+              PrimaryServiceInitializedStateData(
+                maybeNextTick,
+                furtherActivationTicks,
+                simulationStart,
+                valueClass,
+                source,
+              )
+
+            Success(initializedStateData, maybeNextTick)
```

Comment:
We should start at tick 0 then, right? 

```suggestion
            Success(initializedStateData, Some(0L))
```

(This also makes more sense when looking at the case before this one, where we fail because we only have data _after_ simulation start.)